### DISCUSSION TITLE
[New Feature] Debuffs on weapons

### DISCUSSION THREAD

**Discussion by CCTiberius** [Vinifera-Developers/Vinifera#1510] [Feature Requests]
### Description:
The ability for a weapon to apply X debuff to the target for Y seconds. Some commonly wanted ones I have seen are the ability to slow the opponent down without outright stunning them, the ability to lower their damage, make them inaccurate, or make them more susceptible to damage.

### Possible Implementation:
The easiest and most-user friendly way I imagine would be to pre-define the most requested and obvious ones like the ones above, and make something like INI keys along the lines of `AppliesDebuff=Slow` and `DebuffDuration=10` with the number being ticks.

**Comment by Crimsonum** [comment]
It's perhaps worth noting that the example slow-down effect already exists in Firestorm (`LimpetFactor`). Perhaps that code can be expanded/altered so it doesn't consume the attacker in the process unless that attacker is specifically a limpet drone.
